<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>专辑封面管理器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cover_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="text">
     <string>单击列表项目以浏览封面...</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="music_listWidget">
    <property name="geometry">
     <rect>
      <x>1108</x>
      <y>29</y>
      <width>249</width>
      <height>701</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="viewfld_pushButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>700</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="about_pushButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>730</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>431</width>
      <height>165</height>
     </rect>
    </property>
    <property name="title">
     <string>标签信息</string>
    </property>
    <widget class="QPushButton" name="search_pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_pushButton">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>130</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>嵌入当前专辑封面</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>411</width>
       <height>99</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>歌曲名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="music_lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>歌手：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="singer_lineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>专辑：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="al_lineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>封面尺寸：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="size_label">
        <property name="text">
         <string>N\A</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="cover_listWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>635</width>
      <height>721</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>635</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="path_label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择浏览文件夹！</string>
    </property>
   </widget>
   <widget class="QLabel" name="count_label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>740</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>封面来自：网易云音乐</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>封面预览：</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_path_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>670</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>music_listWidget</tabstop>
  <tabstop>cover_listWidget</tabstop>
  <tabstop>music_lineEdit</tabstop>
  <tabstop>singer_lineEdit</tabstop>
  <tabstop>al_lineEdit</tabstop>
  <tabstop>save_pushButton</tabstop>
  <tabstop>search_pushButton</tabstop>
  <tabstop>viewfld_pushButton</tabstop>
  <tabstop>about_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
